packages/gestalt/src/ScrollableContainer.test.js

```diff
@@ -0,0 +1,25 @@
+// @flow strict
+import React from 'react';
+import { create } from 'react-test-renderer';
+import ScrollableContainer from './ScrollableContainer.js';
+import Box from './Box.js';
+
+describe('ScrollableContainer', () => {
+  it('renders', () => {
+    const tree = create(
+      <ScrollableContainer>
+        <Box />
+      </ScrollableContainer>,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('ScrollableContainer correctly sets height and overflow props', () => {
+    const tree = create(
+      <ScrollableContainer height={200} overflow="scroll">
+        <Box />
+      </ScrollableContainer>,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
Rather than just relying on the snapshot, consider looking specifically at the classes and styles being applied

Reply:
All tests rely on the snapshot... we might want to discuss this.